src/sentry/runner/commands/backup.py

```diff
@@ -228,7 +228,8 @@ def import_(src):
     """CLI command wrapping the `exec_import` functionality."""
 
     try:
-        with transaction.atomic():
+        # Import / export only works in monolith mode with a consolidated db.
+        with transaction.atomic("default"):
```

Comment:
Can't remove it -- it's actually needed unfortunately, since the foreign keys are enforced once the transaction completes but will fail if each object is saved discretely.